app/serializers/initial_state_serializer.rb

```diff
@@ -126,10 +131,15 @@ def default_meta_store
       trends_as_landing_page: Setting.trends_as_landing_page,
       trends_enabled: Setting.trends,
       version: instance_presenter.version,
+      visible_reactions: Setting.visible_reactions,
```

Comment:
The most minor nitpick in the history of this pull request: This line should probably be moved to *after* `terms_of_service_enabled`, if alphabetical order is to be preserved.

Reply:
Perhaps, though in this case upstream didn't quite follow it either ("t" doesn't come after "v").